samples/peripheral/lpuart/boards/nrf52dk_nrf52832.overlay

```diff
@@ -12,6 +12,7 @@
 		req-pin = <24>;
 		rdy-pin = <25>;
 	};
+	status = "okay";
```

Comment:
I think this is not needed. We actually take it for granted that `uart0` is enabled for all boards.

Reply:
ok

Reply:
Done